docker-compose.yml

```diff
@@ -3,6 +3,8 @@ services:
   mongo:
     image: mongo
     container_name: mongo
+    environment:
```

Comment:
Note that changes in this file are from the other PR I have open (since I had branched off of that PR branch): https://github.com/FightPandemics/FightPandemics/pull/237